translations/base-en.yaml

```diff
@@ -461,6 +461,11 @@ dialogs:
         missingMods: Missing Mods
         newMods: Newly installed Mods
 
+    applyModsWarning:
+        title: Warning
+        desc: >-
+            The game will reload to apply your changes to the mods.
```

Comment:
strictly speaking, you don't change the mods (or mod options), you change whether they're active